core/src/main/scalajvm/scalapb/zio_grpc/Server.scala

```diff
@@ -147,42 +143,42 @@ object ServerLayer {
 }
 
 object ManagedServer {
-  def fromBuilder(builder: => ServerBuilder[_]): ZManaged[Any, Throwable, Server.Service] =
+  def fromBuilder(builder: => ServerBuilder[_]): ZIO[Scope, Throwable, Server.Service] =
     fromServiceList(builder, ServiceList)
 
   def fromService[R1, S1](builder: => ServerBuilder[_], s1: S1)(implicit
       bs: ZBindableService[R1, S1]
-  ): ZManaged[R1, Throwable, Server.Service] =
+  ): ZIO[R1 with Scope, Throwable, Server.Service] =
     fromServiceList(builder, ServiceList.add(s1))
 
   def fromServices[R1, S1, R2, S2](builder: => ServerBuilder[_], s1: S1, s2: S2)(implicit
       bs1: ZBindableService[R1, S1],
       bs2: ZBindableService[R2, S2]
-  ): ZManaged[R1 with R2, Throwable, Server.Service] =
+  ): ZIO[R1 with R2 with Scope, Throwable, Server.Service] =
     fromServiceList(builder, ServiceList.add(s1).add(s2))
 
   def fromServices[R1, S1, R2, S2, R3, S3](builder: => ServerBuilder[_], s1: S1, s2: S2, s3: S3)(implicit
       bs1: ZBindableService[R1, S1],
       bs2: ZBindableService[R2, S2],
       bs3: ZBindableService[R3, S3]
-  ): ZManaged[R1 with R2 with R3, Throwable, Server.Service] =
-    fromServiceList(builder, ServiceList.add(s1).add(s2).add(s3))
+  ): ZIO[R1 with R2 with R3 with Scope, Throwable, Server.Service] =
+    fromServiceList[R1 with R2 with R3](builder, ServiceList.add(s1).add(s2).add(s3))
 
   def fromServiceList[R](
       builder: => ServerBuilder[_],
       services: URIO[R, List[ServerServiceDefinition]]
-  ): ZManaged[R, Throwable, Server.Service] = fromServiceList(builder, services.toManaged)
+  ): ZIO[R with Scope, Throwable, Server.Service] = fromServiceList(builder, services)
```

Comment:
@adamgfraser the function is just calling itself recursevly, shouldn't it 
be 
```fromServiceListScoped[R](builder, services) ``` ?